JitsiConference.ts

```diff
@@ -2160,12 +2290,17 @@ export default class JitsiConference {
      *
      * @param {JitsiRemoteTrack} removedTrack - The track that was removed.
      */
-    onRemoteTrackRemoved(removedTrack) {
+    onRemoteTrackRemoved(removedTrack: JitsiRemoteTrack): void {
         this.getParticipants().forEach(participant => {
             const tracks = participant.getTracks();
 
             for (let i = 0; i < tracks.length; i++) {
-                if (tracks[i] === removedTrack) {
+                // Compare by a unique property to avoid type incompatibility
```

Comment:
This doesn't make sense. Both are track instances so we know in advance they do have those functions defined.

Reply:
👍  Please drop the comment too.